repo/packages/Z/zeppelin/1/config.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "zeppelin": {
+            "type": "object",
+            "properties": {
+                "framework-name": {
```

Comment:
Because the dcos-ui requires this. Otherwise, you won't see icons and health checks.